tests/llmcompressor/recipe/test_recipe.py

```diff
@@ -3,10 +3,8 @@
 import pytest
 import yaml
 
-from llmcompressor.modifiers import Modifier
```

Comment:
can we add a new test for a list of modifiers?

Reply:
Added Ben!